### DISCUSSION TITLE
Game crashes when accessing the leaderboard on song select screen

### DISCUSSION THREAD

**Discussion by MeguMario** [ppy/osu#15422] [Q&A] [answered]
As in the title, game crashes when accessing the leaderboard on song select screen
Currently unknown if it also occur on computer. Tested on Android for now

Build: 2021.1103.0

Demonstration video: https://streamable.com/9oq0hk

[runtime.log](https://github.com/ppy/osu/files/7462878/runtime.log)
[performance.log](https://github.com/ppy/osu/files/7462882/performance.log)
[network.log](https://github.com/ppy/osu/files/7462886/network.log)
[database.log](https://github.com/ppy/osu/files/7462887/database.log)

**Comment by peppy** [comment, marked as answer]
Will be fixed by https://github.com/ppy/osu/pull/15425.
